akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala

```diff
@@ -308,14 +308,15 @@ abstract class ActorContextSpec extends TypedAkkaSpec {
   import ActorContextSpec._
 
   val config = ConfigFactory.parseString(
-    """|akka {
-     |  loglevel = WARNING
-     |  actor.debug {
-     |    lifecycle = off
-     |    autoreceive = off
-     |  }
-     |  typed.loggers = ["akka.testkit.typed.TestEventListener"]
-     |}""".stripMargin)
+    """
+     akka {
+       loglevel = WARNING
+       loggers = ["akka.testkit.TestEventListener"]
```

Comment:
So we don't need a specific typed `TestEventListener` anymore? Neat :)

Reply:
Maybe we should have, but the current one was more of a hassle than an improvement.